assets/src/components/detoursTable.tsx

```diff
@@ -1,49 +1,107 @@
-import React from "react"
+import React, { ComponentProps } from "react"
 import { Table } from "react-bootstrap"
 import { RoutePill } from "./routePill"
 import { useCurrentTimeSeconds } from "../hooks/useCurrentTime"
 import { timeAgoLabel } from "../util/dateTime"
 import { SimpleDetour } from "../models/detour"
 
+type DetoursTableStatus = "draft" | "active" | "closed"
+
 interface DetoursTableProps {
-  data: SimpleDetour[]
+  data: SimpleDetour[] | null
+  status: DetoursTableStatus
 }
 
-export const DetoursTable = ({ data }: DetoursTableProps) => {
+export const DetoursTable = ({ data, status }: DetoursTableProps) => (
+  <Table hover={data !== null} className="c-detours-table">
+    <thead className="u-hide-for-mobile">
+      <tr>
+        <th className="px-3 py-4">Route and direction</th>
+        <th className="px-3 py-4 u-hide-for-mobile">Starting Intersection</th>
+        <th className="px-3 py-4 u-hide-for-mobile">On detour since</th>
+      </tr>
+    </thead>
+    <tbody>
+      <DetourRows data={data} status={status} />
+    </tbody>
+  </Table>
+)
```

Comment:
nit: I'd prefer if we didn't abstract this yet, and avoid any prop drilling that we'd need to do by having this intermediate component.

Reply:
Which part do you not want to abstract? I can't quite tell, but I think Github might be rendering this weird.

Reply:
I was thinking that `<DetourRows/>` would be inlined here

Reply:
[Done](https://github.com/mbta/skate/pull/2777/commits/ddf0a40aa51ec4ab37342511f600c93ab8c85dd1)